apps/web/src/hooks/workflowChannelPreferences/types.ts

```diff
@@ -0,0 +1,11 @@
+import { ChannelTypeEnum } from '@novu/shared';
+
+type ChannelPreference = {
+  defaultValue: boolean;
+  readOnly: boolean;
+};
+
+export type WorkflowChannelPreferences = {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** Can we import these instead of re-defining?‏